<robot name="plaque-tubes">

  <link name="base_link">  
    <visual>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
        <mesh filename="package://agimus_demos/meshes/plaque-tubes.obj" scale="0.001 0.001 0.001"/>
      </geometry>
    </visual>
    <collision>
      <origin xyz="0 0 0" rpy="0 0 0" />
      <geometry>
        <mesh filename="package://agimus_demos/meshes/plaque-tubes.obj"  scale="0.001 0.001 0.001"/>
      </geometry>
    </collision>
    <inertial>
      <mass value="2.2"/>
      <origin xyz="0 0 0" rpy="0 0 0"/>
      <inertia ixx="0.1"  ixy="0"  ixz="0" iyy="0.1" iyz="0" izz="0.1" />
    </inertial>
  </link>

</robot>
